src/server/utils/cdnCache.js

```diff
@@ -26,15 +26,18 @@ export const oneAppModuleCachePath = path.join(oneAppDirectoryPath, cacheFileNam
 export const showCacheInfo = async () => {
   try {
     const stats = await fsPromises.stat(oneAppModuleCachePath);
+    if (stats.size > 1024 * 1024 * 10) {
+      console.warn(chalk.bold.redBright('🚨🚨🚨: Cache file size is more than 10MB. 🚨🚨🚨'));
```

Comment:
Isn't this pretty common for large tenancies?

Reply:
Yah I don't think this warning is necessary, or should be up at 300mb

Reply:
i guess ill just delete it.

Reply:
@smackfu the cache only keeps one copy of any file for a module. It doesn't not keep multiple versions. So if you only work on one tenancy, your cache size is pretty 'stable' because new versions of your modules replace old versions